arm_controller:
    shoulder_pan_joint:
      type: velocity_controllers/JointVelocityController
      joint: shoulder_pan_joint
      pid: {p: 10.0, i: 0, d: 100.0}
    shoulder_lift_joint:
      type: velocity_controllers/JointVelocityController
      joint: shoulder_lift_joint
      pid: {p: 10.0, i: 0, d: 100.0}
    elbow_joint:
      type: velocity_controllers/JointVelocityController
      joint: elbow_joint
      pid: {p: 10.0, i: 0, d: 100.0}
    wrist_1_joint:
      type: velocity_controllers/JointVelocityController
      joint: wrist_1_joint
      pid: {p: 10.0, i: 0, d: 100.0}
    wrist_2_joint:
      type: velocity_controllers/JointVelocityController
      joint: wrist_2_joint
      pid: {p: 10.0, i: 0, d: 100.0}
    wrist_3_joint:
      type: velocity_controllers/JointVelocityController
      joint: wrist_3_joint
      pid: {p: 10.0, i: 0, d: 100.0}